### PR TITLE
BIDS import creates group if it did not exist

### PR DESCRIPTION
**Pull request by nandorm**

### CODE REVIEW

File: flywheel_cli/commands/import_bids.py [hunk 1]
```diff
@@ -17,6 +17,17 @@ def import_bids(args):
     import flywheel_bids.upload_bids
 
     fw = create_flywheel_client()
+    is_group = None
```
Comment (ambrussimon) [inline flywheel_cli/commands/import_bids.py]: There's no need for a pre-defined default, you could just create the group in the except block.

File: flywheel_cli/commands/import_bids.py [hunk 2]
```diff
@@ -17,6 +17,17 @@ def import_bids(args):
...
+    try:
+        is_group = fw.lookup(args.group)
+    except:
```
Comment (ambrussimon) [inline flywheel_cli/commands/import_bids.py]: Broad `except`.

File: flywheel_cli/commands/import_bids.py [hunk 3]
```diff
@@ -17,6 +17,17 @@ def import_bids(args):
...
+        pass
+    if not is_group:
+        group = {
```
Comment (ambrussimon) [inline flywheel_cli/commands/import_bids.py]: I suggest going more compact here (ie.: single line). Also, _id should come before label.

File: flywheel_cli/commands/import_bids.py (hunk 2 above)
Comment (ambrussimon) [reply flywheel_cli/commands/import_bids.py]: Bare, actually...

File: flywheel_cli/commands/import_bids.py [hunk 4]
```diff
@@ -17,6 +18,11 @@ def import_bids(args):
     import flywheel_bids.upload_bids
 
     fw = create_flywheel_client()
+    try:
+        fw.lookup(args.group)
+    except flywheel.ApiException:
+        group = {"_id": args.group, "label": args.group}
```
Comment (ehlertjd) [inline flywheel_cli/commands/import_bids.py]: Please check the exception status code, and only perform this on a 404.
Additionally, print a message saying that you're creating the group.

**Pull request merged**